### ISSUE TITLE
Exception from sub adminGetOne

### ISSUE THREAD

**Issue by payner35** [orionjs/orion#267]
Seem to be getting a lot of this.. on every Admin collection view

in this case here the collection is called.. "Company"

 Exception from sub adminGetOne.company id JAhPQGFhDnxpTcsqz Error: Match error: Expected string, got object
I20150813-19:33:20.320(4)?     at checkSubtree (packages/check/match.js:159:1)
I20150813-19:33:20.320(4)?     at check (packages/check/match.js:32:1)
I20150813-19:33:20.320(4)?     at [object Object].Meteor.publish.is_auto (packages/orionjs:collections/publications.js:4:1)
I20150813-19:33:20.320(4)?     at packages/matb33:collection-hooks/collection-hooks.js:218:1
I20150813-19:33:20.320(4)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20150813-19:33:20.320(4)?     at [object Object]._handler (packages/matb33:collection-hooks/collection-hooks.js:217:1)
I20150813-19:33:20.321(4)?     at maybeAuditArgumentChecks (packages/ddp/livedata_server.js:1617:1)
I20150813-19:33:20.322(4)?     at [object Object]._.extend._runHandler (packages/ddp/livedata_server.js:950:1)
I20150813-19:33:20.323(4)?     at [object Object]._.extend._startSubscription (packages/ddp/livedata_server.js:769:1)
I20150813-19:33:20.323(4)?     at [object Object]._.extend.protocol_handlers.sub (packages/ddp/livedata_server.js:582:1)
I20150813-19:33:20.323(4)? Sanitized and reported to the client as: Match failed [400]

**Comment by Ajaxsoap** [comment]
Me too, I'm getting this message with all of my relationship. Does your "Company" collection has any relationship with other collection?

**Comment by kevohagan** [comment]
Yep I also confirm gettting this error since updating orion versions. In my case its the relationship between image file and posts collection

```
I20150814-12:00:07.390(2)? Exception from sub adminGetOne.posts id kvSTRmq3xZyyEF5PL Error: Match error: Expected string, got object
I20150814-12:00:07.391(2)?     at checkSubtree (packages/check/match.js:159:1)
I20150814-12:00:07.391(2)?     at check (packages/check/match.js:32:1)
I20150814-12:00:07.391(2)?     at [object Object].Meteor.publish.is_auto (packages/orionjs:collections/publications.js:4:1)
I20150814-12:00:07.391(2)?     at packages/matb33:collection-hooks/collection-hooks.js:218:1
I20150814-12:00:07.391(2)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20150814-12:00:07.391(2)?     at [object Object]._handler (packages/matb33:collection-hooks/collection-hooks.js:217:1)
I20150814-12:00:07.391(2)?     at maybeAuditArgumentChecks (packages/ddp/livedata_server.js:1617:1)
I20150814-12:00:07.392(2)?     at [object Object]._.extend._runHandler (packages/ddp/livedata_server.js:950:1)
I20150814-12:00:07.392(2)?     at [object Object]._.extend._startSubscription (packages/ddp/livedata_server.js:769:1)
I20150814-12:00:07.392(2)?     at [object Object]._.extend.protocol_handlers.sub (packages/ddp/livedata_server.js:582:1)
I20150814-12:00:07.392(2)? Sanitized and reported to the client as: Match failed [400]

```

**Comment by specik** [comment]
I'm having the same problem. Can't figure this one out. Seems to be breaking my post edit view, some of the field values are not being populated in the form even though they exist in the DB.

**Comment by kevohagan** [comment]
I dont understand why this is getting called on the collection view : adminGetOne should be called only on a collection item view, not on the main one no?

That is why adminGetOne check fails since the _id is null (object)

where does this get called @nicolaslopezj @timfam ?

**Comment by nicolaslopezj** [comment]
It's called when it has relations, to show the title field instead of the id

**Comment by nicolaslopezj** [comment]
Is this fixed with ```collections@1.4.2```?
